businessRules/donations.js

```diff
@@ -1,54 +1,59 @@
 const { v4 } = require('uuid');
 const { createDonation } = require('../models/donations');
-const { getAllIdeas } = require('../models/ideas');
-const { getIdeas } = require('../utils/storage');
+const { validateIdea } = require('../validations/ideas');
+const { validateSum } = require('../validations/donations.js');
+const { getUser } = require('./users');
+const { getIdea } = require('./ideas');
 
-function createDonation(firstName, sum, userId, ideaId) {
-    let ideas = getIdeas();
-    const idea = ideas.find((idea) => ideaId === idea.id);
+function createAnonymousUser(firstName, userId) {
+    userId = v4();
```

Comment:
The `createAnonymousUser` function here gets `userId` parameter and then immediately overwrites its value. What's the purpose of passing `userId` to this function, if the function will overwrite that `userId`?

Reply:
Yeah, I've already fixed that, trying to make that function work. )